README.md

```diff
@@ -66,11 +66,11 @@ That's it - the telemetry data is now sent to the locally running Opentelemetry
 
 In order to send traces directly to Splunk Observability Cloud, you need to:
 
-1. Set `OTEL_TRACES_EXPORTER` to `"jaeger-thrift-splunk"` to use the Jaeger exporter.
-2. Set `OTEL_EXPORTER_JAEGER_ENDPOINT` to
-   `https://ingest.<realm>.signalfx.com/v2/trace` where `realm` is your
-   Splunk APM realm (for example, `https://ingest.us0.signalfx.com/v2/trace`).
-3. Set the `SPLUNK_ACCESS_TOKEN` to your Splunk Observability Cloud [access token](https://docs.splunk.com/Observability/admin/authentication-tokens/api-access-tokens.html).
+1. Set `OTEL_EXPORTER_OTLP_ENDPOINT` to
+   `https://ingest.<realm>.signalfx.com/v2/trace/otlp` where `<realm>` is your
+   Splunk APM realm (for example, `https://ingest.us0.signalfx.com/v2/trace/otlp`).
+1. Set the `SPLUNK_ACCESS_TOKEN` to your Splunk Observability Cloud [access token](https://docs.splunk.com/Observability/admin/authentication-tokens/api-access-tokens.html).
+
```

Comment:
Shouldn't we support ``SPLUNK_REALM`` here, so that the endpoint is modified automatically as in the other distros? These are, in fact, our current instructions here: https://docs.splunk.com/Observability/gdi/get-data-in/application/nodejs/instrumentation/instrument-nodejs-application.html#send-data-directly-to-observability-cloud

+CC @pellared

Reply:
As far as I understood Rauno will add the OTLP HTTP exporter which will make this possible. Let's re-review after the addition, you're on point :smile:

Reply:
Yep. I noticed, that we don't bundle ingest-compatible OTLP exporter for now. I planned to add that in v2 branch, but if I'm going to remove the docs here it's probably best to also add everything required for the "correct way".